proto/serverless/instrumentation/v1/dev_mode.proto

```diff
@@ -17,10 +17,24 @@ message DevModePayload {
   // The lambda request id where this payload originated from
   string request_id = 3;
 
+  // Extracted Lambda Telemetry API data
+  optional LambdaTelemetry telemetry = 6;
+
   oneof payload {
     // The set of lambda traces that were generated via an internal extension
     TracePayload trace = 4;
     // The req or response data from the instrumented lambda function
     RequestResponse request_response = 5;
   }
 }
+
+// Lambda Telemetry API data. This data is only available for lambda functions that
+// have access to the telemetry API so it will not be included in all regions.
+message LambdaTelemetry {
+  // Init duration in milliseconds as reported by the metrics on the platform.initReport event
+  optional uint32 init_duration_ms = 1;
+  // Internal runtime duration in milliseconds as reported by the metrics on the platform.runtimeDone event
+  optional uint32 internal_duration_ms = 2;
```

Comment:
Wouldn't it be better to follow AWS naming, and name it `runtime_duration_ms` (`internal` suggests to me it's some AWS internal processing and not necessarily handler work)

Reply:
Yeah good point 👍 I was thinking internal since we were using that phrase in other places but I think changing to runtime could avoid some confusion.